src/definitions/modules/slider.js

```diff
@@ -273,6 +282,9 @@ $.fn.slider = function(parameters) {
               $(document).on('mousemove' + eventNamespace, module.event.move);
               $(document).on('mouseup' + eventNamespace, module.event.up);
             }
+          },
+          windowEvents: function() {
+            $window.on('resize' + eventNamespace, module.event.resize);
```

Comment:
Please add the appropriate unbind event in `module.unbind.events()`

```
$window.off('resize' + eventNamespace);

Reply:
Oh right, didn't think about that ! Thanks !